client/layout/sidebar/footer.jsx

```diff
@@ -4,29 +4,17 @@
  * External dependencies
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
 
-/**
- * Internal dependencies
- */
-import config from 'config';
-import HappychatButton from 'components/happychat/button';
-import InlineHelp from 'blocks/inline-help';
-import hasActiveHappychatSession from 'state/happychat/selectors/has-active-happychat-session';
-
-const SidebarFooter = ( { children, isHappychatButtonVisible } ) => (
-	<div className="sidebar__footer">
-		{ children }
-		<InlineHelp />
-		{ isHappychatButtonVisible &&
-			config.isEnabled( 'happychat' ) && (
-				<HappychatButton className="sidebar__footer-chat" allowMobileRedirect />
-			) }
-	</div>
-);
-
-const mapState = state => ( { isHappychatButtonVisible: hasActiveHappychatSession( state ) } );
+class SidebarFooter extends Component {
```

Comment:
Was there a rationale behind upgrading this to a "full" component vs. keeping it stateless?

Reply:
Whoops! No, I think it's a relict from a temporary experiment. Great catch!